proto/services/task_svc/v1/patient_svc.proto

```diff
@@ -9,43 +9,117 @@ option go_package = "gen/proto/services/task-svc";
 //
 
 message CreatePatientRequest {
-	string human_readable_identifier = 1; // @gotags: validate:"required"
+    string human_readable_identifier = 1; // @gotags: validate:"required"
+    string notes = 2;
 }
 
 message CreatePatientResponse {
-	string id = 1;
+    string id = 1;
 }
 
 //
 // GetPatient
 //
 
 message GetPatientRequest {
-	string id = 1; // @gotags: validate:"uuid4"
+    string id = 1; // @gotags: validate:"uuid4"
 }
 
 message GetPatientResponse {
-	string id = 1; // @gotags: validate:"uuid4"
-	string human_readable_identifier = 2; // @gotags: validate:"required"
+    string id = 1; // @gotags: validate:"uuid4"
+    string human_readable_identifier = 2; // @gotags: validate:"required"
+    string notes = 3;
```

Comment:
validate tags are missing in a bunch of places

Reply:
Validate tags for `string notes = 3;`? This should already be handled by proto. Can you mark your concerns?